packages/@n8n/permissions/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+const sharedOptions = require('@n8n_io/eslint-config/shared');
+
+/**
+ * @type {import('@types/eslint').ESLint.ConfigData}
+ */
+module.exports = {
+	extends: ['@n8n_io/eslint-config/base'],
+
+	...sharedOptions(__dirname),
+
+	rules: {
+		'@typescript-eslint/consistent-type-imports': 'error',
```

Comment:
This is already the rule value in `@n8n_io/eslint-config/base`?

Reply:
I copied the rules from `@n8n/oauth-client` so it looks like that needs fixed too.